libraries/AP_InertialSensor/AP_InertialSensor_Invensense.cpp

```diff
@@ -587,7 +587,9 @@ bool AP_InertialSensor_Invensense::_accumulate(uint8_t *samples, uint8_t n_sampl
                 return false;
             } else {
                 if (!hal.scheduler->in_expected_delay()) {
+                    #ifndef HAL_ESP32_BOARD_NAME
```

Comment:
is there any particular reason to require additional setting in hwdef for these to go away?
could these be HAL_BOARD_NAME == "ESP32" instead?
really we are getting kind of convoluted 
HAL_BOARD_NAME
HAL_ESP32_BOARD_NAME
HAL_BOARD_TYPE
HAL_BOARD_SUBTYPE
used in various places to answer the "IS_THIS_ESP32" question

/rant

Reply:
probably just add a case in the debug() function definition on start to nuke it on esp. That will be simpler that adding ifndef everywhere

Reply:
fixed in latest version pf pr - i took @khancyr's suggestion on this one, and made the debug() call esp32 aware